applications/models.py

```diff
@@ -111,6 +111,7 @@ class Application(models.Model):
 
     # Giv me a resume here!
     resume = models.FileField(upload_to='resumes', null=True, blank=True)
+    cvs_edition = models.BooleanField(default=False)
```

Comment:
@oriolclosa This field name is pretty awful. Can you rename it to cv_authorized, please? (remember to update the form.py as well)

Reply:
Iep, sure, I'll do it later today (it's 1am here).